src/layer/aerialway.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+import * as Color from "../constants/color.js";
+
+// Exponent base for inter-zoom interpolation
+let aerialwayExp = 1.2;
+
+let lineWidth = [
+  "interpolate",
+  ["exponential", aerialwayExp],
+  ["zoom"],
+  12,
+  1.2,
+  14.9999,
+  2.4,
+  15,
+  1.2,
+  16,
+  1.2,
+  20,
+  2.4,
+];
+let lineGapWidth = [
+  "interpolate",
+  ["exponential", aerialwayExp],
+  ["zoom"],
+  12,
+  0,
+  15,
+  0,
+  16,
+  1,
+  20,
+  16,
+];
+
+let casingLineWidth = [
+  "interpolate",
+  ["exponential", aerialwayExp],
+  ["zoom"],
+  12,
+  2,
+  15.9999,
+  4,
+  16,
+  2,
+  20,
+  4,
+];
+let casingLineGapWidth = [
+  "interpolate",
+  ["exponential", aerialwayExp],
+  ["zoom"],
+  12,
+  0,
+  16,
+  0,
+  20,
+  14.4,
+];
+
+export const lift = {
+  id: "lift",
+  type: "line",
+  paint: {
+    "line-color": Color.aerialwayLine,
+    "line-width": lineWidth,
+    "line-gap-width": lineGapWidth,
+  },
+  filter: [
+    "all",
+    ["==", ["get", "class"], "aerialway"],
+    [
+      "in",
+      ["get", "subclass"],
+      ["literal", ["chair_lift", "cable_car", "gondola", "mixed_lift"]],
+    ],
+  ],
+  layout: {
+    visibility: "visible",
+  },
+  source: "openmaptiles",
+  "source-layer": "transportation",
+};
+
+export const liftCasing = {
+  id: "lift_casing",
+  type: "line",
+  paint: {
+    "line-color": Color.backgroundFill,
+    "line-width": casingLineWidth,
+    "line-gap-width": casingLineGapWidth,
```

Comment:
The legend doesn’t correctly implement `line-gap-width`, making this legend entry look misleading: #770.

[<img src="https://user-images.githubusercontent.com/1231218/217360236-8c1d799c-8e5d-46cc-aa25-5682cd673795.png" width="400" alt="Roosevelt Island Tramway">](https://zelonewolf.github.io/openstreetmap-americana/#map=17.82/40.761363/-73.962493)